### PR TITLE
adding lake to the outdoors

### PR DESCRIPTION
**Pull request by CaydenYo**

### CODE REVIEW

File: 01-python/outdoors/lake.py [hunk 1]
```diff
@@ -1,3 +1,16 @@
+import math
 def draw_lake():
-    print("lake not found")
+    r = 5
```
Comment (gallettilance) [inline 01-python/outdoors/lake.py]: what is `r`? please use descriptive variable names

File: 01-python/outdoors/lake.py [hunk 2]
```diff
@@ -1,3 +1,16 @@
+import math
 def draw_lake():
```
Comment (gallettilance) [inline 01-python/outdoors/lake.py]: r should be a parameter

File: 01-python/outdoors/lake.py (hunk 1 above)
Comment (CaydenYo) [reply 01-python/outdoors/lake.py]: sorry about that, i got it, thx!

Comment (gallettilance) [reply 01-python/outdoors/lake.py]: it's fine :) please update your PR with a fix when you have time

**Pull request closed without merging**